### ISSUE TITLE
ReferenceError: Promise is not defined

### ISSUE THREAD

**Issue by jaseeey** [niftylettuce/node-email-templates#152]
Whilst running a test, I came across a particular error:

```bash
Unhandled rejection ReferenceError: Promise is not defined
    at renderFile (/home/ubuntu/jbideploy/node_modules/email-templates/lib/util.js:52:21)
    at /home/ubuntu/jbideploy/node_modules/email-templates/lib/email-template.js:121:64
    at tryCatcher (/home/ubuntu/jbideploy/node_modules/email-templates/node_modules/bluebird/js/main/util.js:26:23)
    at Promise._settlePromiseFromHandler (/home/ubuntu/jbideploy/node_modules/email-templates/node_modules/bluebird/js/main/promise.js:507:31)
    at Promise._settlePromiseAt (/home/ubuntu/jbideploy/node_modules/email-templates/node_modules/bluebird/js/main/promise.js:581:18)
    at Promise._settlePromises (/home/ubuntu/jbideploy/node_modules/email-templates/node_modules/bluebird/js/main/promise.js:697:14)
    at Async._drainQueue (/home/ubuntu/jbideploy/node_modules/email-templates/node_modules/bluebird/js/main/async.js:123:16)
    at Async._drainQueues (/home/ubuntu/jbideploy/node_modules/email-templates/node_modules/bluebird/js/main/async.js:133:10)
    at Async.drainQueues (/home/ubuntu/jbideploy/node_modules/email-templates/node_modules/bluebird/js/main/async.js:15:14)
    at process._tickCallback (node.js:419:13)
```

Looking at lib/util.js, it appears there is a reference to `Promise`, although through the rest of your package it uses `P`. Is this an error, or am I missing something?

**Comment by damiandennis** [comment]
Promise is native now, you might be running an out of date nodejs installation, but looks like he is mixing bluebird with the native promise. would be best to swap the Promise with P for backward compatibility.
also if you installed nodejs via apt-get you are probably running 0.10.*. Install node via nvm to save you some hassle with using newer versions.
https://github.com/creationix/nvm

**Comment by jaseeey** [comment]
This error was only encountered under a certain condition, however it would be ideal to have this fixed for consistency in the package.

**Comment by jeduan** [comment]
Fixed in e97a76f

**Comment by jaseeey** [comment]
@jeduan Thanks for advising, hadn't really been on top of this issue as it wasn't a blocker.
